types/src/v0/traits.rs

```diff
@@ -804,16 +837,30 @@ impl<P: SequencerPersistence> Storage<SeqTypes> for Arc<P> {
             )
             .await
     }
-
     async fn append_proposal(
         &self,
         proposal: &Proposal<SeqTypes, QuorumProposal<SeqTypes>>,
     ) -> anyhow::Result<()> {
         (**self)
-            .append_quorum_proposal(&convert_proposal(proposal.clone()))
+            .append_quorum_proposal2(&convert_proposal(proposal.clone()))
             .await
     }
 
+    async fn append_proposal2(
+        &self,
+        proposal: &Proposal<SeqTypes, QuorumProposal2<SeqTypes>>,
+    ) -> anyhow::Result<()> {
+        // TODO: this is a bug in hotshot with makes with_epoch = true
```

Comment:
Is this comment still valid? It's not clear to me what we need to do. Fix a bug in hotshot? Do some changes when a bug in hotshot is fixed? Or?